src/policies/list/components/PolicyEnactmentsDrawer/PolicyEnactmentsDrawer.tsx

```diff
@@ -42,28 +45,28 @@ const PolicyEnactmentsDrawer: React.FC<PolicyEnactmentsDrawerProps> = ({
 
   const tabsData = useMemo(
     () => [
-      { title: t('Failing'), icon: <RedExclamationCircleIcon />, enactments: failingEnactments },
+      { title: 'Failing', icon: <RedExclamationCircleIcon />, enactments: failingEnactments },
```

Comment:
Shouldn't we better use `EnactmentStatuses.Failing` here instead, for the `title`?

Reply:
Yes right